src/convertToObject.js

```diff
@@ -3,15 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
+  const result = {};
+  const newString = sourceString.split(';').map(elem => elem.split(': '));
```

Comment:
Is it a string? I thought that array 🤔

```suggestion
  const betterName = sourceString.split(';').map(elem => elem.split(': '));
```